src/helpers/color.ts

```diff
@@ -26,9 +26,8 @@ export function colorNumberString(color: number): Color {
   return toHex(color.toString(16).padStart(6, "0"));
 }
 
-let colorIndex = 0;
-export function getNextColor() {
-  colorIndex = ++colorIndex % colors.length;
+export function getNextColor(index: number) {
+  const colorIndex = ++index % colors.length;
```

Comment:
since you increment the index beforehand, you will skip the first element of `colors`  which should be orange.